### PR TITLE
feat: oa-verify v4 (alpha)

### PR DESCRIPTION
**Pull request by HJunyuan**
## What does this PR do?
- chore: upgrade to node 18
- ci: create alpha release based on `alpha` branch
- feat: introduce oa-verify v4 (alpha) to verify VCs that are signed with DIDs
  - Document Integrity
  - Document Status
  - Issuer Identity

## Caveat
- Verification of documents issued onto the document store are not available yet - on hold as not crucial at the moment
- Tests are still WIP

## Todo
- [x] Bump `@govtechsg/open-attestation` to the alpha release (/Open-Attestation/open-attestation/pull/254)

### CODE REVIEW

**Comment by HJunyuan** [comment]
FYI build is currently failing as `@govtechsg/open-attestation` needs to be bumped first

**Review by cavacado — COMMENTED** [review]
should be fine, seems like many of the file changes are prettier errors.

why are the tests wip though, needs the oa pr to go in first?

File: package.json [hunk 1]
```diff
@@ -75,6 +74,19 @@
...
+      {
+        "name": "alpha",
+        "prerelease": true
+      },
+      {
+        "name": "beta",
+        "prerelease": true
+      }
+    ]
+  },
```
[the remark this answers is not in the document]
Comment (HJunyuan) [reply package.json:89]: Will move this to it's own `.releaserc` file

File: package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
   "dependencies": {
     "@govtechsg/dnsprove": "^2.6.1",
     "@govtechsg/document-store": "^2.2.3",
-    "@govtechsg/open-attestation": "^6.2.0",
+    "@govtechsg/open-attestation": "../open-attestation",
```
[the remark this answers is not in the document]
Comment (HJunyuan) [reply package.json:30]: This line is temporary. We will change it to the alpha version once it's merged and published

**Comment by HJunyuan** [comment]
> seems like many of the file changes are prettier errors.

To see the changes without the lint fixes: https://github.com/Open-Attestation/oa-verify/pull/259/files/d2ecc625b925549989bb72fa7583ea05c030e5d8

> why are the tests wip though, needs the oa pr to go in first?

To save time, the first alpha release will not include a majority of tests. Same for the `open-attestation` repo. We can work on them in the subsequent alpha versions

**Review by cavacado — APPROVED** [review]
same as the other one

**Pull request merged**